### PR TITLE
Feature/eco places component

### PR DESCRIPTION
**Pull request by HryshkoAnton**

### CODE REVIEW

File: src/app/component/user/components/profile/eco-places/eco-places.component.ts [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
-  ngOnInit() {}
+  ngOnInit() {
+    this.getEcoPlaces();
+  }
 
-  public getEcoPlaces(): Array<string> {
-    return this.ecoPlaces.slice(0, 3);
+  public getEcoPlaces(): EcoPlaces[] {
+    return this.profileService.getEcoPlaces().subscribe(
+      (success) => {
```
Comment (RomanDeveloperAcc) [inline src/app/component/user/components/profile/eco-places/eco-places.component.ts:22]: If you have only one assignment or one fuction call, you don't need to write it in separate lines, so please, rewrite next three lines in one line.

File: src/app/component/user/components/profile/eco-places/eco-places.component.ts [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
+        this.ecoPlaces = success;
+      },
+      (error) => {
```
Comment (RomanDeveloperAcc) [inline src/app/component/user/components/profile/eco-places/eco-places.component.ts:25]: Same as line 22

File: src/app/component/user/components/profile/profile-service/profile.service.ts [hunk 3]
```diff
@@ -35,4 +35,8 @@ export class ProfileService {
     return this.http.get<object>(`${this.backEnd}user/isOnline/${this.userId}/`);
   }
 
+  public getEcoPlaces(): any {
```
Comment (RomanDeveloperAcc) [inline src/app/component/user/components/profile/profile-service/profile.service.ts:38]: It is bad to leave the 'any' type, so please, define the type of the function

File: src/app/component/user/components/profile/eco-places/eco-places.component.ts (hunk 1 above)
Comment (HryshkoAnton) [reply src/app/component/user/components/profile/eco-places/eco-places.component.ts]: done

File: src/app/component/user/components/profile/eco-places/eco-places.component.ts (hunk 2 above)
Comment (HryshkoAnton) [reply src/app/component/user/components/profile/eco-places/eco-places.component.ts]: done

File: src/app/component/user/components/profile/profile-service/profile.service.ts (hunk 3 above)
Comment (HryshkoAnton) [reply src/app/component/user/components/profile/profile-service/profile.service.ts]: done

**Comment by mplesha** [comment]
conflict

**Pull request merged**